### PR TITLE
Make review requirements more lenient.

### PR DESCRIPTION
**Pull request by chrisdembia**
I've attempted to update the review guidelines based on our discussion yesterday. Please feel free to edit the document directly.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2523)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ When you are ready to make a PR, please adhere to the following guidelines:
...
 
-6. Any other changes to the code require review by at least two members of the Dev Team. If the pull request involves adding a new class or performing a major object/algorithm refactor, one of these reviewers must be an Owner. The Owners and Dev Team are Teams within the opensim-org GitHub organization. The original author may NOT merge the pull request.
+6. Any other changes to the code require review by one member of the Dev Team, and manual testing by one member of the Dev Team. If the pull request involves adding a new class or performing a major object/algorithm refactor, one of these reviewers must be an Owner. The first reviewer determines if a second reviewer is required for the pull request to be accepted; you may also suggest whether the pull request should require one or two reviewers. The Owners and Dev Team are Teams within the opensim-org GitHub organization. You or the reviewers may merge the pull request once the reviewers accept the pull request.
```
Comment (nickbianco) [inline CONTRIBUTING.md]: > You or the reviewers may merge the pull request once the reviewers accept the pull request.

And when the CI tests pass.

**Comment by jenhicks** [comment]
I agree with @nickbianco 's suggestion. Otherwise, looks good.

**Comment by aseth1** [comment]
Let's make the CI tests must all pass a separate item/criterion for merging PRs since it applies to all types of PRs not just more significant PRs that `6.` addresses.

**Comment by jimmyDunne** [comment]
What happens when the CI test fail that has nothing to do with the PR? I have seen CI fails when we are editing comments and it is some completely unrelated class/test?

**Comment by jenhicks** [comment]
Add a bullet point: CI must be run for all changes, except Matlab tests and examples, non-Doxygen markdown files, and non-Doxygen comments. CI tests must pass before merge, unless the author and reviewer(s) deem the failures unrelated to the code change. 

Ayman will make this change to the markdown and give a few days for others to comment before merging. OKed by @aymanhab @chrisdembia @carmichaelong @jenhicks @jimmyDunne

**Comment by jenhicks** [comment]
@aseth1 @tkuchida Please let us know if you have any feedback.

**Comment by jenhicks** [comment]
Thanks @tkuchida . Sounds good. @aymanhab will add the links in.

@aseth1 Can you let us know if you have any feedback by Wednesday (Aug 21)?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ When you are ready to make a PR, please adhere to the following guidelines:
...
+6. Any other changes to the code require review by one member of the Dev Team, and manual testing by one member of the Dev Team. If the pull request involves adding a new class or performing a major object/algorithm refactor, one of these reviewers must be an Owner. The first reviewer determines if a second reviewer is required for the pull request to be accepted; you may also suggest whether the pull request should require one or two reviewers. The Owners and Dev Team are Teams within the opensim-org GitHub organization. You or the reviewers may merge the pull request once the reviewers accept the pull request. Owners can be determined from  [this list](https://github.com/orgs/opensim-org/people); Dev Team members are listed [here](https://github.com/orgs/opensim-org/teams/dev-team/members).
```
Comment (jenhicks) [inline CONTRIBUTING.md]: Looks good, but I would switch the order of the last two sentences.

**Comment by aymanhab** [comment]
Switch order per @jenhicks comments, should be ready to merge.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ When you are ready to make a PR, please adhere to the following guidelines:
...
+6. Any other changes to the code require review by one member of the Dev Team, and manual testing by one member of the Dev Team. If the pull request involves adding a new class or performing a major object/algorithm refactor, one of these reviewers must be an Owner. The first reviewer determines if a second reviewer is required for the pull request to be accepted; you may also suggest whether the pull request should require one or two reviewers. The Owners and Dev Team are Teams within the opensim-org GitHub organization. You or the reviewers may merge the pull request once the reviewers accept the pull request. Dev Team members are listed [here](https://github.com/orgs/opensim-org/teams/dev-team/members). Owners can be determined from [this list](https://github.com/orgs/opensim-org/people).
```
Comment (jenhicks) [inline CONTRIBUTING.md]: This is what I mean:

The Owners and Dev Team are Teams within the opensim-org GitHub organization. Dev Team members are listed [here](https://github.com/orgs/opensim-org/teams/dev-team/members). Owners can be determined from [this list](https://github.com/orgs/opensim-org/people). You or the reviewers may merge the pull request once the reviewers accept the pull request.

**Comment by jenhicks** [comment]
@aymanhab Will make the update and then we can merge.

**Comment by aymanhab** [comment]
@jenhicks Please let me know if I captured your intent of rearranging the text, and if so feel free to merge.

**Comment by jenhicks** [comment]
Thanks! Please feel free to merge.

**Pull request merged**